cscs-checks/mch/cuda_stress_test.py

```diff
@@ -26,6 +30,9 @@ def __init__(self):
             'time': sn.extractsingle(r'Timing: (\S+)', self.stdout, 1, float)
         }
         self.reference = {
+            'arolla:cn': {
+                'time': (2.25, None, 0.05)
```

Comment:
This number is a bit strange. Let's discuss this.

Reply:
See my comment on reference values above

Reply:
Please remove the reference.